core-bundle/src/Resources/contao/pages/PageRegular.php

```diff
@@ -204,14 +206,14 @@ protected function prepare($objPage)
 
 		// Set the page title and description AFTER the modules have been generated
 		$this->Template->mainTitle = $objPage->rootPageTitle;
-		$this->Template->pageTitle = htmlspecialchars($responseContext->getTitle());
+		$this->Template->pageTitle = htmlspecialchars($responseContext->get(HtmlHeadBag::class)->getTitle());
```

Comment:
don't we need a `null`-check here (and the lines below)?

Reply:
Well, we set the response context ourselves in this method and we know that it has an `HtmlHeadBag`.